src/service/dialCodeService.js

```diff
@@ -946,6 +946,42 @@ function releaseDialCode(req, response) {
   ])
 }
 
+/**
+ * This function helps to update the status of the given process
+ * @param {type} req
+ * @param {type} response
+ * @returns {object} return response object with http status
+ */
+function updateProcessIdStatus (req, response) {
+  var data = {}
+  data.body = req.body
+  data.processId = req.params.processId
+  var rspObj = req.rspObj
+  // Adding objectData in telemetryData object
+  if (rspObj.telemetryData) {
+    rspObj.telemetryData.object = utilsService.getObjectData(data.processId, 'updateProcessIdStatus', '', {})
+  }
+
+  if (!data.processId) {
+    LOG.error(utilsService.getLoggerData(rspObj, 'ERROR', filename, 'updateProcessIdStatus',
+      'Error due to required params are missing', {
```

Comment:
Instead of using hardcoded strings. Isn't there a message strings file?

Reply:
No